web/src/components/Organization/OrganizationPickListsCell/OrganizationPickListsCell.tsx

```diff
@@ -6,19 +6,30 @@ import type {
 import { Label, SelectField, HiddenField } from '@redwoodjs/forms'
 import type { CellSuccessProps, CellFailureProps } from '@redwoodjs/web'
 
+import { useAuth } from 'src/auth'
+
+export const beforeQuery = () => {
+  // https://redwoodjs.com/docs/7.4/cells#beforequery
```

Comment:
Using `useAuth()` in `beforeQuery` gives the following eslint error: `React Hook "useAuth" is called in function "beforeQuery" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use"  react-hooks/rules-of-hooks`. Both `useAuth` and `beforeQuery` hooks are coming from Redwood, so renaming them is not really feasible.